Atala PRISM/2.9.md

```diff
@@ -0,0 +1,68 @@
+# Atala PRISM v2.9
+
+February, 2024 release of Atala PRISM.
+
+## Components
+
+### Platform
+
+* [Open Enterprise Agent v1.25.0](https://github.com/hyperledger-labs/open-enterprise-agent/releases/tag/prism-agent-v1.25.0)
+* [PRISM Node v2.2.1](https://github.com/input-output-hk/atala-prism/releases/tag/v2.2.1)
+* [PRISM Mediator v0.10.2](https://github.com/input-output-hk/atala-prism-mediator/releases/tag/prism-mediator-v0.10.2)
+
+### SDKs
+
+* [Apollo Crypto Library v1.2.10](https://github.com/input-output-hk/atala-prism-apollo/releases/tag/v1.2.10)
+* [Wallet SDK Swift v4.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/4.0.0)
+* [Wallet SDK TypeScript v4.0.2](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/releases/tag/v4.0.1)
+* [Wallet SDK KMM v2.7.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/releases/tag/v2.7.0)
+
+### Documentation
+
+* [Documentation Portal v1.76.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.76.0)
+
+## Changelog
+
+**Updated components:**
+
+- Open Enterprise Agent: v1.21.0 -> v1.25.0
+- Documentation Portal: v1.69.0 -> v1.76.0
+- Apollo Crypto Library: v1.1.1 -> v1.2.10
+- Wallet SDK KMM: v2.6.0 -> v2.7.0
+- Wallet SDK TypeScript: v3.1.0 -> v4.0.2
+- PRISM Mediator: no changes
+- Wallet SDK Swift: v3.6.0 -> v4.0.0
+- PRISM Node: no changes
+
+**New Features:**
+
+- **Open Enterprise Agent**:
+  - Added JWT auth support for agent-admin role
+  - Upgraded ZIO http client to improve performance
+  - Improved documentation of the OpenAPI specification
+- **Documentation Portal**:
+  - Build SDK TypeScript documentation using GitHub submodules
```

Comment:
I believe we should mention the Quick Start Guide was reviewed and it can be executed from start to finish. I am not sure with which versions of each component though. In the QSG, it states: Agent v1.18.0. This seems to be unchanged. But I know Shailesh and Ben were running it so maybe this needs to be updated.